configure

```diff
@@ -392,6 +392,11 @@ parser.add_option('--xcode',
     dest='use_xcode',
     help='generate build files for use with xcode')
 
+parser.add_option('--ninja',
+    action='store_true',
+    dest='use_ninja',
+    help='generate a Ninja based build system')
```

Comment:
Sounds like it produces the build system itself, could we clarify this to be like the xcode one? (Also per @jbergstroem's comment)